src/test/lambda/models/samLambdaRuntime.test.ts

```diff
@@ -123,3 +124,18 @@ describe('runtimes', function () {
         ])
     })
 })
+
+describe('getNodeMajorVersion()', () => {
+    it(`extracts major version from 'nodejs{VERSION}.x'`, () => {
+        for (let i = 1; i < 1000; i++) {
+            const versionString = `nodejs${i}.x`
+            const version = getNodeMajorVersion(versionString)
```

Comment:
My suggestion about a "range" test was to check the `version` result.

    assert(version > 0 && version < 999)

Reply:
Done!